pkg/engine/params_k8s.go

```diff
@@ -244,13 +244,6 @@ func assignKubernetesParameters(properties *api.Properties, parametersMap params
 			addValue(parametersMap, "kubernetesEndpoint", properties.HostedMasterProfile.FQDN)
 		}
 
-		// GPU nodes need docker-engine as the container runtime
-		if properties.HasNSeriesSKU() {
```

Comment:
should we also remove from https://github.com/Azure/aks-engine/blob/master/parts/k8s/kubernetesparams.t#L322?

Reply:
done